pages/common/csvpy.md

```diff
@@ -0,0 +1,12 @@
+# csvpy
+
+> Loads a CSV file into a Python shell.
```

Comment:
Yeah, I struggled a bit trying to find an explanation of what this does that wasn't too wordy. :stuck_out_tongue: